content/zh/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -390,239 +400,36 @@ and in the logs to monitor the state of the auditing subsystem.
 - `apiserver_audit_error_total` 在暴露时由于发生错误而被丢弃的事件的数量。
 
 <!--
-## Setup for multiple API servers
-
-If you're extending the Kubernetes API with the [aggregation layer][kube-aggregator], you can also
-set up audit logging for the aggregated apiserver. To do this, pass the configuration options in the
-same format as described above to the aggregated apiserver and set up the log ingesting pipeline
-to pick up audit logs. Different apiservers can have different audit configurations and different
-audit policies.
--->
-## 多 API 服务器的配置
-
-如果你通过[聚合层](/zh/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/)
-对 Kubernetes API 进行扩展，那么你也可以为聚合的 API 服务器设置审计日志。
-想要这么做，你需要以上述的格式给聚合的 API 服务器配置参数，并且配置日志管道以采用审计日志。
-不同的 API 服务器可以配置不同的审计配置和策略。
-
-<!--
-## Log Collector Examples
-
-### Use fluentd to collect and distribute audit events from log file
-
-[Fluentd](https://www.fluentd.org/) is an open source data collector for unified logging layer.
-In this example, we will use fluentd to split audit events by different namespaces.
--->
-## 日志收集器示例
-
-### 使用 fluentd 从日志文件中选择并且分发审计日志
```

Comment:
why is all of this commented out?

Reply:
IT was removed from English version possibly because fluentd is 3rd-party content.